README.md

```diff
@@ -52,9 +52,7 @@ Every PR must be prepared using the following commands:
 ```bash
 cd $GOPATH/src/github.com/go-clang/gen
 scripts/switch-clang-version.sh 3.4
-make install
-make test
-make lint
+make test-all
```

Comment:
I am wondering if we should add additional make targets to run things inside the container, or should we advice a contributor to open a bash using the build image?